pkg/sql/set_cluster_setting.go

```diff
@@ -193,6 +195,18 @@ func (p *planner) getAndValidateTypedClusterSetting(
 }
 
 func (n *setClusterSettingNode) startExec(params runParams) error {
+
+	if strings.Contains(n.name, "sql.defaults") {
+		params.p.BufferClientNotice(
+			params.ctx,
+			errors.WithHintf(
+				pgnotice.Newf("Setting global default %s`;", n.name),
+				"use the `ALTER ROLE ... SET` syntax to control session variable defaults at a finer-grained level. See: %s",
+				docs.URL("https://www.cockroachlabs.com/docs/stable/alter-role.html#set-default-session-variable-values-for-a-role"),
```

Comment:
this should just be `alter-role.html#set-default-session-variable-values-for-a-role`